apps/portals/elportal/src/config/synapseConfigs/handleFilesToParticipants.tsx

```diff
@@ -42,7 +42,9 @@ export const handleSelectedFilesToParticipants = async (
   event: CustomControlCallbackData,
 ) => {
   // add filter for files perspective, to show participants associated to the selected files only.
-  const idColIndex = event.data?.columnModels?.findIndex(cm => cm.name === 'id')
+  const idColIndex = event.queryMetadata?.columnModels?.findIndex(
+    cm => cm.name === 'id',
+  )
   // if selected rows were defined, then get the ids from the event selected rows.  Otherwise, get ALL ids (across all pages)
```

Comment:
`data` was changed to `queryMetadata` to make it more clear that row data is not included.